app/models/concerns/user_access_key_overrides.rb

```diff
@@ -20,6 +20,9 @@ def valid_password?(password)
 
   def password=(new_password)
     @password = new_password
+  end
+
+  def encrypt_and_digest_password!
     return if @password.blank?
```

Comment:
should we set the `encrypted_password_digest` to empty string or something if `@password.blank?`